configure.py

```diff
@@ -1234,6 +1234,8 @@ def find_ninja():
        Json2Code('api/api-doc/authorization_cache.json'),
        'api/raft.cc',
        Json2Code('api/api-doc/raft.json'),
+       Json2Code('api/api-doc/cql_server_test.json'),
```

Comment:
@Jadw1 hi Michal, could you please

1.  update `api/CMakeLists.cmake` accordingly, namely, to add  `api-doc/cql_server_test.json` right after `api-doc/config.json`.
2. and add `cql_server_test.cc` right after `config.cc`

? thanks.